<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="pixmap">
        <pixmap resource="res.qrc">:/pic/title.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openCloseBtn">
       <property name="text">
        <string>Closed</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/pic/shut_down_grayed.png</normaloff>:/pic/shut_down_grayed.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="RealTimeMonitorWidget" name="realTimePage">
      <attribute name="title">
       <string>RealTime Monitor</string>
      </attribute>
     </widget>
     <widget class="HistoryMonitorWidget" name="historyPage">
      <attribute name="title">
       <string>History Monitor</string>
      </attribute>
     </widget>
     <widget class="ControlSettingWidget" name="ctrlSettingPage">
      <attribute name="title">
       <string>Control Setting</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RealTimeMonitorWidget</class>
   <extends>QWidget</extends>
   <header>RealTimeMonitorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HistoryMonitorWidget</class>
   <extends>QWidget</extends>
   <header>HistoryMonitorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlSettingWidget</class>
   <extends>QWidget</extends>
   <header>ControlSettingWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
